core/src/main/resources/hudson/model/DownloadService/footer.jelly

```diff
@@ -29,7 +29,7 @@ THE SOFTWARE.
 
   This file is pulled into the layout.jelly
 -->
-<?jelly escape-by-default='false'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  ${it.generateFragment()}
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+  <j:out value="${it.generateFragment()}"/>
```

Comment:
I don't really understand what a change like this adds to a file like this one.

- Pro: When someone adds content to the file, people need to remember to escape or perform a change like this.
- Con: Adds complexity, and potential `st:out` / `j:out` confusion.

Reply:
It was actually for the Pro. After that commit normally all the Jelly views will be escaped by default. Specific behavior will be explicitly written with the usage of `j:out` for non-escaping content.

I think the confusion is just my fault, when I wrote this PR I was not totally aware of the difference between `st:out` and `j:out`. After your comments, there will be no other `st:out` usage in the PR.

Reply:
Well, the confusion remains if someone doesn't remember offhand which is which.

Not a big deal, just wondered.